src/app/embed/embed.component.ts

```diff
@@ -15,11 +16,13 @@ const PYM_CHILD_ID_PARAM = 'childId';
   styleUrls: ['embed.component.css'],
   providers: [MergeAdapter, QSDAdapter, DraperAdapter, FeedAdapter],
   template: `
-    <play-share-modal *ngIf="showShareModal" (close)="hideModal()">
-    </play-share-modal>
+    <play-share-modal *ngIf="showShareModal" (close)="hideModal()"></play-share-modal>
     <play-player [feedArtworkUrl]="feedArtworkUrl" [audioUrl]="audioUrl" [title]="title" [subtitle]="subtitle"
       [subscribeUrl]="subscribeUrl" [subscribeTarget]="subscribeTarget" [artworkUrl]="artworkUrl" (share)="showModal()"
       [showPlaylist]="showPlaylist" [episodes]="episodes">
+      <ng-template let-dismiss="dismiss">
```

Comment:
I'm guessing this will allow you to dismiss the modal from a link/button/whatever inside the template content.

Reply:
yes.